server/myenv/Lib/site-packages/django/conf/__init__.py

```
@@ -0,0 +1,272 @@
+        self.__dict__["_deleted"] = set()
+        self.default_settings = default_settings
+
+    def __getattr__(self, name):
+        if not name.isupper() or name in self._deleted:
+            raise AttributeError
+        return getattr(self.default_settings, name)
+
+    def __setattr__(self, name, value):
+        self._deleted.discard(name)
+        if name == "FORMS_URLFIELD_ASSUME_HTTPS":
+            warnings.warn(
```

Ruff (0.8.2)

No explicit `stacklevel` keyword argument found

(B028)